frontend/cypress/plugins/index.js

```diff
@@ -65,6 +65,20 @@ module.exports = (on, _config) => {
       const wm = spawn("./cypress/support/wiremock/start-wiremock.sh", [
         stubDir,
       ]);
+      let scriptOutput = "";
```

Comment:
do we need this or is this for debugging purposes? 🤔

Reply:
I put it in for debugging purposes. I think it may be helpful in the future, but I can just remove it and take note of this in my documentation about wiremock!